docusaurus/docs/get-started/set-up-development-environment.mdx

```diff
@@ -19,7 +19,7 @@ The `create-plugin` tool includes a development environment featuring [Docker](h
 
 :::info
 
-It's not necessary to [sign a plugin](../publish-a-plugin/sign-a-plugin.md) during development. The that is scaffolded with `@grafana/create-plugin` will load the plugin without a signature.
+It's not necessary to [sign a plugin](../publish-a-plugin/sign-a-plugin.md) during development. The development environment that is scaffolded with `@grafana/create-plugin` will load the plugin without a signature.
```

Comment:
We have a statement in https://github.com/grafana/plugin-tools/pull/462/ which is more complete since it also references how it works, probably best to keep consistent: 

> It's not necessary to sign a plugin during development. The [Docker development environment](../get-started/set-up-development-environment.mdx) that is scaffolded with `@grafana/create-plugin` will load the plugin without a signature. This is because it is configured by default to run in [development mode](https://github.com/grafana/grafana/blob/main/contribute/developer-guide.md#configure-grafana-for-development).